content/posts/shutdown/shutdown.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "shutdown"
+date: 2022-03-13T01:48:05+03:30
+draft: false
+---
+
+<div dir='rtl'>
+
+### فهرست
+
+- [فهرست](#فهرست)
+- [مقدمه](#مقدمه)
+- [خاموش کردن کامپیوتر](#خاموش-کردن-کامپیوتر)
+- [ریبوت کردن کامپیوتر](#ریبوت-کردن-کامپیوتر)
+- [خاموش شدن کامپیوتر در ساعت مشخص](#خاموش-شدن-کامپیوتر-در-ساعت-مشخص)
+- [ریبوت شدن کامپیوتر در ساعت مشخص](#ریبوت-شدن-کامپیوتر-در-ساعت-مشخص)
+- [لغو کردن عملیات](#لغو-کردن-عملیات)
+- [ارسال پیغام به دیگر کاربران قبل از خاموش یا ریبوت شدن](#ارسال-پیغام-به-دیگر-کاربران-قبل-از-خاموش-یا-ریبوت-شدن)
+- [نمایش مجدد عملیات](#نمایش-مجدد-عملیات)
+- [Author or Authors](#author-or-authors)
+</div>
+
+
+
+
+---
+<div dir='rtl'>
+
+### مقدمه
+اگر حوصله ی استفاده از رابط گرافیکی برای خاموش کردن کامپیوتر را ندارید . بهترین میانبر استفاده از خط فرمان است. با استفاده از ابزار
+کامپیوتر خود را خاموش کنید.
+البته بهتر است بدانید که به جز خاموش کردن امکان ریبوت کامپیوتر هم دراین ابزار وجود دارد که با آن آشنا خواهیم شد. shutdown خط فرمان میتوانید به راحتی 
+</div>
+
+
+
+---
+<div dir='rtl'>
+
+### خاموش کردن کامپیوتر
+برای خاموش کردن کامپیوتر از سوییچ h- استفاده کنید. با استفاده از این سوییچ کامپیوتر شما بعد از گذشت یک دقیقه و سی ثانیه خاموش خواهد شد. اگر میخواهید که .کامپیوتر شما در همان لحظه خاموش شود از عبارت now در کنار دستور خود استفاده کنید.
+</div>
+    
+    $ sudo shutdown -h
+    # poweroff scheduled for Sun 2022-03-13 16:00:00 +0330, use 'shutdown -c' to cancel.
+
+    Or 
+    
+    $ sudo shutdown -h now
+
+
+
+---
+<div dir='rtl'>
+
+### ریبوت کردن کامپیوتر
+اگر میخواهید که کامپیوتر شما ریبوت شود از سوییچ r- استفاده کنید. این سوییچ کامپیوتر شما را یک دقیقه و سی ثانیه بعد از تایید عملیات ریبوت میکند.
+اگر میخواهید که کامپیوتر شما در همان لحظه ریبوت شود همانند عملیات خاموش شدن از عبارت now در کنار دستور خود استفاده کنید.
+</div>
+
+    $ sudo shutdown -r
+    # Reboot scheduled for Sun 2022-03-13 15:36:51 +0330, use 'shutdown -c' to cancel.
+
+    Or 
+
+    $ sudo shutdown -r now
+
+
+---
+<div dir='rtl'>
+
+### خاموش شدن کامپیوتر در ساعت مشخص
+اگر میخواستید که کامپیوتر شما در ساعت مشخصی خاموش شود به جای عبارت now از ساعت مشخصی استفاده کنید.
+</div>
+
+    $ sudo shutdown -h 16:44
```

Comment:
grate, but add the other way of doing this too,

by specifying the second to shutdown 

`shutdown 120`